docs/starters.yml

```diff
@@ -4130,3 +4130,27 @@
     - Mobile friendly responsive design made to be customized or leave as is
     - Separate components for everything
     - ...and more
+- url: https://gatsby-starter-zurgbot.netlify.com/
+  repo: https://github.com/zurgbot/gatsby-starter-zurgbot
+  description: The ultimate force of starter awesomeness in the galaxy of Gatsby
+  tags:
+    - Landing Page
+    - Linting
+    - PWA
+    - SEO
```

Comment:
```suggestion
```

Reply:
curious about why this one is invalid given the solution in there to add and manage appropriate meta + the sitemap plugin working there (not to mention the sitemap tag removal change)

Reply:
didn't saw the latter, it's fine then.

Reply:
ah okay cool